frontend/apps/marketing/README.md

```diff
@@ -47,3 +47,22 @@ To build a docker image, run this command in the `frontend` directory:
 ```bash
 docker build -f apps/marketing/Dockerfile .
 ```
+
+## OpenTelemetry
```

Comment:
Being new to OTEL, is there value in adding some documentation around the "frontend/src/otel" stuff? Either as comments, a local README, or something in this README? I don't really know what the processor and sampler are for (I also haven't spent much effort trying to find out)

Reply:
Yep - added the comments